src/Microsoft.ML.AutoML/Experiment/Runners/RunnerUtil.cs

```diff
@@ -25,21 +27,21 @@ public static (ModelContainer model, TMetrics metrics, Exception exception, doub
             try
             {
                 var estimator = pipeline.ToEstimator(trainData, validData);
-                var model = estimator.Fit(trainData);
+                _estimatorModel = estimator.Fit(trainData);
```

Comment:
Why is it necessary to rename the model? It is causing needless changes elsewhere in the code.

Reply:
Here I renamed `model` to `estimatorModel` as I think it does a better job of explaining the model that is returned by a estimator.Fit() call. In addition `TrainAndScorePipeline `returned `ModelContainer` variable is named `model`, and I thought that was confusing to have 2 variables named model (one an `ITransformer`, the other is a `ModelContainer`).

Here the options are renaming the returned ModelContainer (which results in the need to modify this name in other places where  `TrainAndScorePipeline ` is called), or renaming the model that is returned by the estimator.Fit() function (which results in the obvious need to rename it in other parts of this code). @justinormont what do you think?

Reply:
No strong opinions on the naming. Is there a convention in ML․NET? 

Generally I think of a fit "pipeline" as a "model". Though aligning on the ML․NET terminology would be good too.

Terms you'll find in the AutoML code:
* `Recipe`: General layout for how the AutoML will solve the problem at hand
* `Pipeline`: A `recipe` which is fit to the current dataset; for instance handling the N categorical columns in the specific dataset
* `Model`: A trained `pipeline` which can now give predictions

See more: https://patents.google.com/patent/US20190018821A1/en

Reply:
I've removed the `estimatorModel` name, and corrected ModelContainer's name to reflect that it is a `modelContainer `and not just a `model`. However now ModelContainer.Dispose() needs to be called to free these Tensor objects, and I don't know where this should occur. Any ideas?